### ISSUE TITLE
Regular Expression gets recognized as Object

### ISSUE THREAD

**Issue by MichaelFlucher** [iljapostnovs/ui5plugin-linter#50]
**Describe the bug**
I have a simple Code to transform `lower_values` to `camelCase`:
```javascript
sText.replace(/_([a-z])/g, function (g) {
    return g[1].toUpperCase();
});
```

This code gives the following error:
`"beginIndex" param is of type "regexp|string", but provided "object"`

**To Reproduce**
1. Insert Code
2. run linter

**Expected behavior**
Recognize regular expression

**Screenshots**

**Desktop (please complete the following information):**
 - Version of ui5plugin-linter: 0.5.17
 - ui5version used: 1.89.0

**Additional context**

**Project attachment**
If applicable, Project example for issue reproduction

**Comment by iljapostnovs** [comment]
Hi,
Thank you for bug report!
Fixed via v0.5.19, please check if it works for you.

**Comment by MichaelFlucher** [comment]
Thank you, it works 🚀
